rootfs/scheduler/__init__.py

```diff
@@ -667,6 +679,44 @@ def _delete_rc(self, namespace, name):
             error(resp, 'delete ReplicationController "{}" in Namespace "{}"',
                   name, namespace)
 
+    def _healthcheck(self, controller, path='/', port=80, delay=30, timeout=1):
+        # FIXME this logic ideally should live higher up
+        if controller['spec']['selector']['type'] not in ['web', 'cmd']:
```

Comment:
Actually i don't think it would be worth it right now to do this change. `_deploy_app` calls a `deploy` which in turn calls `create` which in turn calls `_create_rc` which in turn calls `_healthcheck`

More inclined to fix that all up when #88 starts and for now have something that works. And also have more k8s tests in place first :P